### PR TITLE
Remove all unused fixtures

### PR DESCRIPTION
**Pull request by seallard**
## Description
This PR removes all unused fixtures.

```
pip install pytest-deadfixtures
pytest --dead-fixtures
```

### Fixed
- Removed all unused fixtures

### This [version](https://semver.org/) is a
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Review by diitaz93 — CHANGES_REQUESTED** [review]
Please leave the demultiplex fixtures as I am using them in my PR. If they remain unused I will remove them myself :)

**Comment by seallard** [comment]
@diitaz93 Can you check if there are any additional fixtures that you need that I've removed?

File: tests/store/conftest.py [hunk 1]
```diff
@@ -64,18 +64,6 @@ def generate_year_interval(n_entries: int, old_timestamp: dt.datetime) -> List[i
         return output
 
 
-@pytest.fixture(name="application_versions_file")
-def fixture_application_versions_file(fixtures_dir: Path) -> str:
-    """Return application version import file."""
-    return Path(fixtures_dir, "store", "api", "application_versions.xlsx").as_posix()
```
Comment (diitaz93) [inline tests/store/conftest.py:70]: If this file is not used maybe you could consider removing it too

File: tests/store/conftest.py [hunk 2]
```diff
@@ -64,18 +64,6 @@ def generate_year_interval(n_entries: int, old_timestamp: dt.datetime) -> List[i
...
-
-
-@pytest.fixture(name="applications_file")
-def fixture_applications_file(fixtures_dir: Path) -> str:
-    """Return application import file."""
-    return Path(fixtures_dir, "store", "api", "applications.xlsx").as_posix()
```
Comment (diitaz93) [inline tests/store/conftest.py:76]: same here

File: tests/conftest.py [hunk 3]
```diff
@@ -808,12 +790,6 @@ def fixture_bcl2fast2_flow_cell_id(bcl2fastq_flow_cell: FlowCell) -> str:
     return bcl2fastq_flow_cell.id
 
 
-@pytest.fixture(name="dragen_flow_cell_id")
-def fixture_dragen_flow_cell_id(dragen_flow_cell: FlowCell) -> str:
-    """Return flow cell id from dragen flow cell object."""
-    return dragen_flow_cell.id
```
Comment (diitaz93) [inline tests/conftest.py:814]: I would like you to leave this one too 🙂

**Review by diitaz93 — APPROVED** [review]
Looks good on my side 👍 Great initiative

**Comment by seallard** [comment]
@diitaz93 Thanks for checking! Nice 😄

**Pull request merged**